srcpkgs/lcalc/template

```diff
@@ -0,0 +1,45 @@
+# Template file for 'lcalc'
+pkgname=lcalc
+version=1.23
+revision=1
+build_wrksrc="src"
+build_style="gnu-makefile"
+makedepends="pari-devel"
+short_desc="Michael Rubinstein's L-function calculator"
+maintainer="Gonzalo Tornaría <tornaria@cmat.edu.uy>"
+license="GPL-2.0-or-later"
+homepage="https://mirrors.mit.edu/sage/spkg/upstream/lcalc/"
+distfiles="https://mirrors.mit.edu/sage/spkg/upstream/lcalc/lcalc-${version}.tar.bz2"
+checksum=83d4253ec2f38553b21190d6d0c6b71bab7ea14717f6dde5bd18f60775d3cdb0
+patch_args="-Np1"
+nocross=yes
```

Comment:
Cross builds fine after patching `libLfunction.so: g++` to `libLfunction.so: $(CC)`

Reply:
Shouldn't it be `CXX`, since they are using `g++`?